api/admin/client.go

```diff
@@ -70,6 +70,15 @@ func (c *Client) AliasChain(chain, alias string) (bool, error) {
 	return res.Success, err
 }
 
+// GetChainAliases ...
+func (c *Client) GetChainAliases(chain string) (*GetChainAliasesReply, error) {
```

Comment:
To be a little more user friendly + in line with other clients, I think this should return `([]string, error)` rather than `(*GetChainAliasesReply, error)`